collector/fixtures/proc/mdstat

```diff
@@ -1,13 +1,13 @@
 Personalities : [linear] [multipath] [raid0] [raid1] [raid6] [raid5] [raid4] [raid10]
 md3 : active raid6 sda1[8] sdh1[7] sdg1[6] sdf1[5] sde1[11] sdd1[3] sdc1[10] sdb1[9]
       5853468288 blocks super 1.2 level 6, 64k chunk, algorithm 2 [8/8] [UUUUUUUU]
-      
```

Comment:
I think your editor may have changed the fixture here, could you double-check whether these tabs are there in reality and if so, revert those changes?